### DISCUSSION TITLE
[FEATURE]容器里，支持端口号排序

### DISCUSSION THREAD

**Discussion by Alexzzz2008** [1Panel-dev/1Panel#6495] [Ideas]
### 1Panel 版本

1.10.15

### 请描述您的需求或者改进建议

容器里，支持端口号排序

网站比较多， 希望找出空闲的端口号 ，重利用

**Comment by wanghe-fit2cloud** [comment]
感谢反馈！目前有几点需要讨论：

1.	对于单个容器暴露多个端口的情况，处理起来可能会有问题。
2.	容器列表是通过 Docker SDK 获取的，若涉及分页显示，可能会出现显示问题。
